lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
Variables that end with `-difference` are only used to provide a value for `calc()`.
So, instead of `calc(100% - #{$enrollment-details-width-difference})`, should I revert to the old value of `calc(100% - 237px)`?